Riot/Routers/TabRouters/TabBarRouter.swift

```diff
@@ -0,0 +1,109 @@
+// 
+// Copyright 2021 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+import WeakDictionary
+
+class TabBarRouter: NSObject, TabbedRouterType {
+    
+    // MARK: - Properties
+    
+    // MARK: Private
+    
+    private var completions: [UIViewController : () -> Void]
```

Comment:
Did you add `completions` for the future? I mean for the future handling completions like `NavigationRouter` when you remove a tab item.

Reply:
Good catch. I've removed it as I was intended to have completion callbacks but it appears to be useless as of now.